content-warehouse/src/main/java/contentwarehouse/v1/SearchDocuments.java

```diff
@@ -0,0 +1,129 @@
+import com.google.cloud.contentwarehouse.v1.DocumentServiceClient;
+import com.google.cloud.contentwarehouse.v1.DocumentServiceClient.SearchDocumentsPagedResponse;
+import com.google.cloud.contentwarehouse.v1.DocumentServiceSettings;
+import com.google.cloud.contentwarehouse.v1.FileTypeFilter;
+import com.google.cloud.contentwarehouse.v1.FileTypeFilter.FileType;
+import com.google.cloud.contentwarehouse.v1.LocationName;
+import com.google.cloud.contentwarehouse.v1.RequestMetadata;
+import com.google.cloud.contentwarehouse.v1.SearchDocumentsRequest;
+import com.google.cloud.contentwarehouse.v1.SearchDocumentsResponse.MatchingDocument;
+import com.google.cloud.contentwarehouse.v1.UserInfo;
+import com.google.cloud.resourcemanager.v3.Project;
+import com.google.cloud.resourcemanager.v3.ProjectName;
+import com.google.cloud.resourcemanager.v3.ProjectsClient;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class SearchDocuments {
+  public static void searchDocument() throws IOException, 
+        InterruptedException, ExecutionException, TimeoutException { 
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String location = "your-region"; // Format is "us" or "eu".
+    String documentQuery = "your-document-query";
+    String userId = "your-user-id"; // Format is user:<user-id>
+
+    searchDocuments(projectId, location, documentQuery, userId);
+  }
+
+  // Searches all Documents for given Document Query
+  public static void searchDocuments(String projectId, String location,
+        String documentQuery, String userId) throws IOException, InterruptedException,
+          ExecutionException, TimeoutException { 
+    String projectNumber = getProjectNumber(projectId);
+
+    String endpoint = "contentwarehouse.googleapis.com:443";
+    if (!"us".equals(location)) {
+      endpoint = String.format("%s-%s", location, endpoint);
+    }
+
+    DocumentServiceSettings documentServiceSettings = 
+             DocumentServiceSettings.newBuilder().setEndpoint(endpoint).build(); 
+
+    /* Create the Document Service Client 
+     * Initialize client that will be used to send requests. 
+     * This client only needs to be created once, and can be reused for multiple requests. */
+    try (DocumentServiceClient documentServiceClient = 
+            DocumentServiceClient.create(documentServiceSettings)) {  
+
+      /*  The full resource name of the location, e.g.:
+      projects/{project_number}/locations/{location} */
+      String parent = LocationName.format(projectNumber, location);
+
+      // Define RequestMetadata object for context of the user making the API call
+      RequestMetadata requestMetadata = RequestMetadata.newBuilder()
+          .setUserInfo(
+          UserInfo.newBuilder()
+            .setId(userId).build()).build();
```

Comment:
It's good practice to move the build command to a new line to align with the chain hierarchy. Applies to multiple occurrences in this file.

```suggestion
      RequestMetadata requestMetadata = RequestMetadata.newBuilder()
        .setUserInfo(
            UserInfo.newBuilder()
                .setId(userId)
                .build())
        .build();
```